application/config/rest.php

```diff
@@ -604,3 +604,15 @@
 |
 */
 $config['allowed_cors_origins'] = [];
+
+
+/*
+|--------------------------------------------------------------------------
+| Choose method naming convention
+|--------------------------------------------------------------------------
+|
+| Default value is `snake_case`
+| e.g. send_notification_post, sendNotificationPost
```

Comment:
You should add the posible configuration options 
- camelcase
- snake_case

And `snake_case` should be `snakecase` oder `camelcase` should be `camel_case`

So the style of the optins are the same